waveform_editor/options.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import os
+from pathlib import Path
+
+import panel as pn
+import param
+import yaml
+
+logger = logging.getLogger(__name__)
+
+_xdg = os.environ.get("XDG_CONFIG_HOME")
+_config_home = Path(_xdg) if _xdg else Path(os.environ["HOME"]) / ".config"
```

Comment:
`HOME` may be unset, let's use https://docs.python.org/3/library/pathlib.html#pathlib.Path.home
<img width="780" height="211" alt="image" src="https://github.com/user-attachments/assets/37e73ad7-5c6e-4a0c-b2ce-31de6acbd2da" />